source/includes/_tipser-widget.md

```diff
@@ -145,6 +145,58 @@ A complete index of all the supported configuration options can be found in [All
 It is possible to use Tipser Elements in test (staging) environment to be able to do test checkouts. The environment can be activated using the `env` configuration option, as in the snippet, 
 where supported environments are `stage` and `prod`. This configuration option is optional and in case it is missing, `prod` environment will be used. 
 
+## Adding event handlers ##
+Event handlers may be passed as a part of configuration. There is a number of events exposed for developer. Each of the handler may have a different params as arguments of handler. 
+
+There is `eventsHandler` config option available. This is the place where you can add your own handlers for events.
+
+Currently we are supporting `onError` and `onAddToCart` event handlers.
+
+`onError` is being triggered when some error appear (i.e. when adding product to the cart failed) and `onAddToCart` event is being triggered when the product has been added to the cart.
+
+The handler for `onError` gets error argument (see code example). Error argument is object type with properties : `type`, `id`, `message`, `stack`. The most important information is message and type. Currently all errors have `TipserElementError` and message contains the information for the user what went wrong. `Stack` is a typical error stack of js error.
```

Comment:
`The most important information is message and type. Currently all errors have `TipserElementError` and message contains the information for the user what went wrong.` - does it make sense to write that the most important informaton is `type` and then in the next sentence that there is always the same type? :)